apps/server/src/modules/saga/type/saga-step.ts

```diff
@@ -0,0 +1,39 @@
+import { EntityId } from '@shared/domain/types';
+import { StepReport } from './report';
+
+export interface StepType {
+	deleteUserData: { params: { userId: EntityId }; result: StepReport };
+	// add more, e.g.:
+	// deleteUserReference: { params: { userId: EntityId }; result: boolean };
+	// step1: { params: { value: number }; result: string };
+	// step2: { params: { message: string }; result: boolean };
+}
+
+export enum ModuleName {
```

Comment:
This more or less duplicates deletion/domain/types/domain-name.enum.ts
At the very least we should add a comment in both places to be aware we need to maintain it in both locations....

Reply:
I think we can remove deletion/domain/types/domain-name.enum.ts

Reply:
probably, but for now there are plenty of places where is used in the deletion module